elephant/conversion.py

```diff
@@ -152,3 +155,704 @@ def binarize(spiketrain, sampling_rate=None, t_start=None, t_stop=None,
+        Stop time
+
+    Returns
+    -------
+    t_start : quantities.Quantity
+        Starting point calculated from given parameter.
+    """
+    if num_bins is not None and binsize is not None and t_stop is not None:
+        return t_stop.rescale(binsize.units) - num_bins * binsize
+
+
+def calc_tstop(num_bins, binsize, t_start):
+    """
+    Calculates the stop point from given parameter.
+
+    Calculates the stop point :attr:`t_stop` from the three parameter
+    :attr:`t_start`, :attr:`num_bins` and :attr`binsize`.
+
+    Parameters
+    ----------
+    num_bins: int
+        Number of bins
+    binsize: quantities.Quantity
+        Size of bins
+    t_start: quantities.Quantity
+        Start time
+
+    Returns
+    -------
+    t_stop : quantities.Quantity
+        Stoping point calculated from given parameter.
+    """
+    if num_bins is not None and binsize is not None and t_start is not None:
+        return t_start.rescale(binsize.units) + num_bins * binsize
+
+
+def calc_num_bins(binsize, t_start, t_stop):
+    """
+    Calculates the number of bins from given parameter.
+
+    Calculates the number of bins :attr:`num_bins` from the three parameter
+    :attr:`t_start`, :attr:`t_stop` and :attr`binsize`.
+
+    Parameters
+    ----------
+    binsize: quantities.Quantity
+        Size of Bins
+    t_start : quantities.Quantity
+        Start time
+    t_stop: quantities.Quantity
+        Stop time
+
+    Returns
+    -------
+    num_bins : int
+       Number of bins  calculated from given parameter.
+
+    Raises
+    ------
+    ValueError :
+        Raised when :attr:`t_stop` is smaller than :attr:`t_start`".
+
+    """
+    if binsize is not None and t_start is not None and t_stop is not None:
+        if t_stop < t_start:
+            raise ValueError("t_stop (%s) is smaller than t_start (%s)"
+                             % (t_stop, t_start))
+        return int(((t_stop - t_start).rescale(
+            binsize.units) / binsize).magnitude)
+
+
+def calc_binsize(num_bins, t_start, t_stop):
+    """
+    Calculates the stop point from given parameter.
+
+    Calculates the size of bins :attr:`binsize` from the three parameter
+    :attr:`num_bins`, :attr:`t_start` and :attr`t_stop`.
+
+    Parameters
+    ----------
+    num_bins: int
+        Number of bins
+    t_start: quantities.Quantity
+        Start time
+    t_stop
+       Stop time
+
+    Returns
+    -------
+    binsize : quantities.Quantity
+        Size of bins calculated from given parameter.
+
+    Raises
+    ------
+    ValueError :
+        Raised when :attr:`t_stop` is smaller than :attr:`t_start`".
+    """
+
+    if num_bins is not None and t_start is not None and t_stop is not None:
+        if t_stop < t_start:
+            raise ValueError("t_stop (%s) is smaller than t_start (%s)"
+                             % (t_stop, t_start))
+        return (t_stop - t_start) / num_bins
+
+
+def set_start_stop_from_input(spiketrains):
```

Comment:
Yes, this function should be a getter. 
Then again, I do not agree to make this function a non-method. Besides that this is a function, which is used a lot.  In our lab nearly everybody has this kind of function somewhere in his own script. So it would be better to have such functions in a central place then scattered around. That does not mean, that this has to stay in the `conversion` module.  I would prefer it even in `neo`.